### PR TITLE
Added IDE markers for intellij and vscode

### PR DESCRIPTION
**Pull request by midays**

### CODE REVIEW

File: src/tests/intellij/test_advanced_options.py [hunk 1]
```diff
@@ -47,6 +49,7 @@ def test_skip_reports_intellij(configurations, setup_intellij, app_name, analysi
 
 @pytest.mark.parametrize("app_name", ["weblogic_to_eap7_legacy_reports"])
 @pytest.mark.parametrize("ide", ["intellij"])
+@pytest.mark.intellij
```
Comment (nachandr) [inline src/tests/intellij/test_advanced_options.py:52]: You could apply markers at the module level. Maybe, there is a way to apply it at the directory level as well.

[Markers at module level](https://docs.pytest.org/en/7.1.x/example/markers.html#marking-whole-classes-or-modules)

**Review by nachandr — CHANGES_REQUESTED** [review]

File: src/tests/intellij/test_advanced_options.py (hunk 1 above)
Comment (midays) [reply src/tests/intellij/test_advanced_options.py:52]: @nachandr  Thanks for the feedback, according to the documenation applying the markes at module level requires to implement a class, when I did that it showed unrecognized variables errors that required changing how the tests are structured to fix those variable errors, I think it's best to just add it to each test it's not much work

**Review by sshveta — APPROVED** [review]

File: src/tests/vscode/test_analysis.py [hunk 2]
```diff
@@ -8,6 +8,7 @@
 
 @pytest.mark.parametrize("app_name", json.load(open(DATA_DIR + "analysis.json")))
 @pytest.mark.parametrize("ide", ["vscode"])
+@pytest.mark.intellij
```
Comment (nachandr) [inline src/tests/vscode/test_analysis.py]: This has an incorrect marker .

Comment (midays) [reply src/tests/vscode/test_analysis.py]: Fixed

**Review by nachandr — APPROVED** [review]

**Pull request merged**